README.md

```diff
@@ -64,6 +64,63 @@ and other docs.
 <br />
 <!-- END OF README-JOB SECTION -->
 
+## Homebound Fork
+
+This is Homebound's fork of `date-fns`. We forked this repository to extend the functionality of the `businessDate`
+functions. Our plan is to PR these improvements back to the mainstream project. However, in the meantime, we have
+logic to publish a `.patch` file to layer this functionality on top of the upstream `date-fns` package via the
+[`patch-package` project](https://github.com/ds300/patch-package).
+
+### Our Branch
+
+Currently, our code changes live on the [`improved-business-days`](https://github.com/homebound-team/date-fns/tree/improved-business-days)
+branch on our fork. The intention with this branch is that it's well-tested and will be PR'ed against the upstream
+project.
+
+This branch is tested using the same GitHub actions that run on the upstream project. When you push to this branch, it
+does not automatically trigger a new patch publish (see [below](#publishing-the-patch-package) for how to do that).
+
+### Publishing the Patch Package
+
+There are two distinct cases you might want to trigger a new package publish:
+
+1. If you've made changes on the `improved-business-days` branch and want to release them.
+1. If a new version of `date-fns` is released upstream.
+
+In either case, you trigger a release by running the [workflow in CircleCI](https://app.circleci.com/pipelines/github/homebound-team/date-fns?branch=homebound-patch-publish&filter=all). Just pick the last workflow run and click the "Rerun workflow from the start" button.
```

Comment:
As mentioned above, if the manual trigger is confusing, we could change this to automatically trigger on any push to the `improved-business-days` branch.